<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/icons/simpleconvert</normaloff>:/icons/simpleconvert</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="lblFFmpegBinary">
       <property name="text">
        <string>FFMPEG Binary:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editFFmpegBinary"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbOverwriteOutput">
     <property name="text">
      <string>Overwrite output when exists</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowNotificationConverted">
     <property name="text">
      <string>Show notification when converting is done</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
